django/publicmapping/redistricting/management/commands/setup.py

```diff
@@ -47,16 +40,9 @@
 from os.path import exists
 from lxml.etree import parse, XSLT
 from redistricting.config import Utils, SpatialUtils
-from redistricting.models import (
-    Geolevel,
-    Geounit,
-    Subject,
-    Characteristic,
-    LegislativeBody,
-    Plan,
-    ProcessingState,
-    configure_views
-)
+from redistricting.models import (Geolevel, Geounit, Subject, Characteristic,
```

Comment:
I'm personally a fan of how it was before (each item in a list on one line). More lines are taken up but it's trivial to make your editor sort them which makes it easier to read and find what you're looking for.

Reply:
I agree -- this is a `yapf` artifact. If this is the price we pay for having automated formatting then I guess I'm okay with it, but if we can configure `yapf` to do multi-line imports I'd be happier. Not sure whether that's possible -- it looks like it's configurable to some extent but it may be fiddly.